help/_config.yml

```diff
@@ -0,0 +1,6 @@
+defaults:
+  -
```

Comment:
Ah I took this example from the Jekyll documentation https://jekyllrb.com/docs/configuration/front-matter-defaults/ I tried to remove the `-` but breaks the site

Reply:
Oh, gotcha it's an array where the first element is an object. with the `scope` and `values` properties. You could probably add more defaults to the array, each of which could be an object with `scope` and `values`.

Reply:
Thanks for bearing with me.

Reply:
Actually, NAB but the more normal syntax here would be:

```yaml
defaults:
  - scope:
      path: "" # an empty string here means all files in the project
    values:
      layout: default      
```

And then if you wanted to add more defaults it would look like this:

```yaml
defaults:
  - scope:
       ...
     values:
       ...
  - scope:
       ...
     values:
       ...
```

Reply:
ohh that makes sense, I'm going to update it